cmd/podman/containers/create.go

```diff
@@ -64,6 +64,7 @@ func createFlags(cmd *cobra.Command) {
 	if registry.IsRemote() {
 		_ = flags.MarkHidden("conmon-pidfile")
 		_ = flags.MarkHidden("pidfile")
+		_ = flags.MarkHidden("tls-verify")
```

Comment:
I think that's a nop since there is no such flag with that name for create and run.

What we need to do is to 1) create these flags, 2) set them in the options accordingly (may need some optional-bool parsing as in other commands), 3) update the man pages in ./docs/source/markdown, 4) add a test for both commands to exercise the code.

Reply:
thanks for the review @vrothberg, to be honest, I'm not familiar at all with the codebase here and Golang too.
I'll check how this is achieved for `build` subcommand and update that PR.